reference/metadata_tables.rst

```diff
@@ -13,26 +13,35 @@ The pg_dist_partition table stores metadata about which tables in the database a
 +----------------+----------------------+---------------------------------------------------------------------------+
 |      Name      |         Type         |       Description                                                         |
 +================+======================+===========================================================================+
-| logicalrelid   |           oid        | | Distributed table to which this row corresponds. This value references  | 
+| logicalrelid   |         regclass     | | Distributed table to which this row corresponds. This value references  | 
 |                |                      | | the relfilenode column in the pg_class system catalog table.            |
 +----------------+----------------------+---------------------------------------------------------------------------+   
-|  partmethod    |            char      | | The method used for partitioning / distribution. The values of this     |
+|  partmethod    |         char         | | The method used for partitioning / distribution. The values of this     |
 |                |                      | | column corresponding to different distribution methods are :-           |
 |                |                      | | append: 'a'                                                             |
 |                |                      | | hash: 'h'                                                               |
 +----------------+----------------------+---------------------------------------------------------------------------+
-|   partkey      |            text      | | Detailed information about the distribution column including column     |
+|   partkey      |         text         | | Detailed information about the distribution column including column     |
 |                |                      | | number, type and other relevant information.                            |
 +----------------+----------------------+---------------------------------------------------------------------------+
+|   colocationid |         integer      | | Co-location group to which this table belongs. Tables in the same group |
+|                |                      | | allow co-located joins and distributed rollups among other              |
+|                |                      | | optimizations. This value references the colocationid column in the     |
+|                |                      | | pg_dist_colocation table.                                               |
++----------------+----------------------+---------------------------------------------------------------------------+
+|   repmodel     |         char         | | The method used for data replication. The values of this column         |
+|                |                      | | corresponding to different replication methods are :-                   |   
```

Comment:
I actually don't know what the `:-` is all about. I see that git blame shows my name on lines with it but I think that is just because I imported many of the sections into this repo or moved entire files around which attributed me.